networkx/algorithms/smallworld.py

```diff
@@ -148,20 +148,21 @@ def lattice_reference(G, niter=1, D=None, connectivity=True, seed=None):
     Returns
     -------
     G : graph
-       The latticized graph.
+        The latticized graph.
 
     Notes
     -----
-    The implementation is adapted from the algorithm by Sporns et al. which is
-    inspired from the original work from Maslov and Sneppen (2002) [2]_.
+    The implementation is adapted from the algorithm by Sporns et al. [1]_
```

Comment:
These sentences have lost their periods.

Reply:
Good catch!  It looks like any period after an underscore was removed.
I'll fix it!  :) Thanks!